FAQ.md

```diff
@@ -1,4 +1,5 @@
 Frequently Asked Questions
+==========================
```

Comment:
Eh, I had to remove that, so that VSCode would generate a nice Table of Contents. Not sure which plugin does that though

Reply:
It's working. Maaagic :-) !